vms/platformvm/block/builder/builder.go

```diff
@@ -319,12 +319,10 @@ func packBlockTxs(
 		return nil, err
 	}
 
-	changes, err := txexecutor.AdvanceTimeTo(backend, stateDiff, timestamp)
+	_, err = txexecutor.AdvanceTimeTo(backend, stateDiff, timestamp)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
nit; I like to keep the scope smaller when possible
```suggestion
	if _, err := txexecutor.AdvanceTimeTo(backend, stateDiff, timestamp); if err != nil {
		return nil, err
	}
```